apps/perf-test/tasks/perf-test.ts

```diff
@@ -275,7 +275,7 @@ module.exports = async function getPerfRegressions() {
     outDir,
     tempDir,
     pageActions: async (page, options) => {
-      page.setDefaultTimeout(60000); // set 60s timeout, default is 30s.
+      page.setDefaultTimeout(0); // set no timeout
```

Comment:
Actually it might be good to just summarize your findings so far here.. at least explain why it's (temporarily) 0.

Reply:
added comments